src/test/resources/application-test.yml

```diff
@@ -134,6 +134,7 @@ url-templates:
       booking-cancelled-event-detail: http://api/events/booking-cancelled/#eventId
       booking-changed-event-detail: http://api/events/booking-changed/#eventId
       application-withdrawn-event-detail: http://api/events/application-withdrawn/#eventId
+      assessment-appealed-event-detail: http://api/events/application-withdrawn/#eventId
```

Comment:
I think we missed this earlier -- looks like we meant to say that the event detail can be found at `http://api/events/assessment-appealed/#eventId`